packages/compass-query-bar/src/components/query-history/index.tsx

```diff
@@ -40,13 +41,15 @@ const QueryHistory = ({
   onUpdateRecentChoosen,
 }: QueryHistoryProps) => {
   const [tab, setTab] = useState<QueryHistoryTab>('recent');
+  const connectionInfoAccess = useConnectionInfoAccess();
 
   useTrackOnChange(
     (track: TrackFunction) => {
+      const connectionInfo = connectionInfoAccess.getCurrentConnectionInfo();
```

Comment:
The same here, missing connectionInfoAccess in dependency?

Reply:
thank you! added a eslint rule to have a reminder next time.

Reply:
amazing and thank you <3